src/getdns/getdns_extra.h.in

```diff
@@ -501,13 +502,45 @@ getdns_context_set_tls_query_padding_blocksize(getdns_context *context, uint16_t
  */
 getdns_return_t
 getdns_context_unset_edns_maximum_udp_payload_size(getdns_context *context);
-/** @}
- */
 
- /**
-  * \addtogroup getdns_context
-  *  @{
-  */
+
+typedef enum getdns_loglevel_type {
+	GETDNS_LOG_EMERG   = 0,
+	GETDNS_LOG_ALERT   = 1,
+	GETDNS_LOG_CRIT    = 2,
+	GETDNS_LOG_ERR     = 3,
+	GETDNS_LOG_WARNING = 4,
+	GETDNS_LOG_NOTICE  = 5,
+	GETDNS_LOG_INFO    = 6,
+	GETDNS_LOG_DEBUG   = 7
+} getdns_loglevel_type;
```

Comment:
It is important, because they are exposed constants and thus part of the API.
I copied the names and order from POSIX syslog:
see http://pubs.opengroup.org/onlinepubs/009695399/basedefs/syslog.h.html